hail/src/main/scala/is/hail/methods/VEP.scala

```diff
@@ -96,12 +98,25 @@ object VEP {
       None
     }
   }
+}
+
+case class VEP(config: String, csq: Boolean, blockSize: Int) extends TableToTableFunction {
+  private val conf = VEP.readConfiguration(HailContext.get.hadoopConf, config)
+  private val vepSignature = conf.vep_json_schema
+
+  override def preservesPartitionCounts: Boolean = false
```

Comment:
I thought VEP just adds row annotations, why would the count of rows per partition change?

Reply:
Sometimes VEP returns malformed variants and we skip them.  If you look in the main body, you'll see a flatMap whose body parses the JSON returned by VEP in a try/catch.